.gitignore

```diff
@@ -1,3 +1,5 @@
 *.beam
 deps
 ebin
+erl_crash.dump
+/rel/gestalt/
```

Comment:
It also seems to generate some trash in `/.rebar/` for me. Should we add it here?